packages/svelte/tests/hydration/samples/input-value-changed/main.svelte

```diff
@@ -0,0 +1,6 @@
+<script>
+	const { browser } = $props();
+	let value = browser ? 'browser' : 'server';
```

Comment:
the prop defined in `_config.js` is `name`, not `browser` — this test makes no sense!

Reply:
Sorry I pushed the PR and was meant to put it into draft before I went out.